HW1/P2/P2b.py

```diff
@@ -1,3 +1,32 @@
+import findspark 
+findspark.init()
 from P2 import *
+from pyspark import SparkContext
+import random
 
-# Your code here
+sc = SparkContext()
+
+#create lists of the range for rows and cols  
+ilist = range(2000)
+jlist = range(2000)
+
+random.shuffle(ilist)
+random.shuffle(jlist)
```

Comment:
I did something similar. However, if the dataset is too large, such that it makes sense to use spark, you might not be able to do the shuffle within python before parallelizing. This is something I'm not sure about. In that case, you'd have to parallelize first and then shuffle. I did not implement it that way myself, but it's just something to think about.